rest-api-spec/src/main/resources/rest-api-spec/test/README.asciidoc

```diff
@@ -316,6 +316,8 @@ Supports also regular expressions with flag X for more readability (accepts whit
                    \d+    \s+  \d{2}:\d{2}:\d{2}  \s+  \d+    \s+  \n  $/
 ....
 
+**Note:** `$body` is used mainly for cat apis, to refer to the last obtained response body as a string, while `''` refers to the parsed representation of the string (parsed into a Map by the java runner for instance).
```

Comment:
I think we should start with the description and then provide an example instead of the other way around. How about a slightly different wording?

```
`$body` is used to refer to the last obtained response body as a string, while `''` refers to the parsed representation (parsed into a Map by the Java runner for instance). Having the raw string response is for example useful when testing cat APIs.
```

Reply:
That's fine for me :)

Reply:
@delvedor  can you please change this in your PR? Then I'm happy to approve. :)